pkg/apis/extensions/v1beta1/types.go

```diff
@@ -450,6 +454,8 @@ type JobConditionType string
 const (
 	// JobComplete means the job has completed its execution.
 	JobComplete JobConditionType = "Complete"
```

Comment:
Confirmed. Failed is also more consistent with pod and volume phases.
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/types.go#L1143